sycl/include/CL/sycl/detail/pi.h

```diff
@@ -44,7 +44,7 @@
 //
 #include "CL/cl.h"
 #define _PI_H_VERSION_MAJOR 5
-#define _PI_H_VERSION_MINOR 7
+#define _PI_H_VERSION_MINOR 8
```

Comment:
Please add a note about what's changed in 5.8 in the log above. Also could you track why version wasn't 6.8 as change-log suggests.

Reply:
Added. It looks like the version bump was missed in https://github.com/intel/llvm/pull/4576, so I merged the two.